src/components/Task/List.jsx

```diff
@@ -3,6 +3,9 @@ import { MdDeleteOutline } from "react-icons/md";
 import { Form } from "react-router-dom";
 
 export default function TaskList({ list }) {
+  const handleComplete = (completed) => {
+    return completed === "true" ? "false" : "true";
```

Comment:
Can we not completed ? false : true
use boolean instead of string in such cases